picard/ui/itemviews.py

```diff
@@ -689,16 +698,21 @@ def update(self, update_tracks=True):
                     item.update(update_album=False)
         if album.errors:
             self.setIcon(0, AlbumItem.icon_error)
+            self.setToolTip(0,"Error")
```

Comment:
Please add a space after the comma (PEP8)
Use `N_()` and `_()` to mark strings for translation where applicable.